class.phpmailer.php

```diff
@@ -3972,6 +3960,28 @@ public static function hasLineLongerThanMax($str)
         return (boolean)preg_match('/^(.{'.(self::MAX_LINE_LENGTH + 2).',})/m', $str);
     }
 
+    /**
+     * If a string contains any "special" characters, double-quote the name,
+     * and escape any double quotes with a backslash.
+     *
+     * @param string $str
+     *
+     * @return string
+     *
+     * @see RFC822 3.4.1
+     */
+    public function quotedString($str)
+    {
+        if (preg_match('/[ ()<>@,;:"\/\[\]?=]/', $str)) {
+            //If the string contains any of these chars, it must be double-quoted
+            //and any double quotes must be escaped with a backslash
+            return '"' . str_replace('"', '\\"', $str) . '"';
+        }
+
+        //Return the string untouched, it doesn't need quoting
```

Comment:
```suggestion
            // If the string contains any of these chars, it must be double-quoted
            // and any double quotes must be escaped with a backslash
            return '"' . str_replace('"', '\\"', $str) . '"';
        }

        // Return the string untouched, it doesn't need quoting
```

Reply:
It’s fine for me either way

Reply:
Joomla coding standards makes no sense in not joomla software this would end in a mix which is more bad then using the original.